apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotificationList.jsx

```diff
@@ -74,16 +74,6 @@ export default class ModerationNotificationList extends Component {
     this.setState({ alert: undefined })
   }
 
-  filterNotifications = () => {
-    return this.state.moderationComments.filter(comment => {
-      return this.state.filterItem.filter === ''
-        ? true
-        : this.state.filterItem.name === 'Pending'
-          ? comment.has_pending_notifications
-          : !comment.has_pending_notifications
-    })
-  }
-
```

Comment:
@kha I removed this, because I found that it is not really needed as the filtering is already done when fetching.. Or did I miss something?